pdns/syncres.cc

```diff
@@ -2983,6 +2983,7 @@ bool SyncRes::doResolveAtThisIP(const std::string& prefix, const DNSName& qname,
       t_sstorage.throttle.throttle(d_now.tv_sec, boost::make_tuple(remoteIP, qname, qtype.getCode()), 60, 3);
       return false;
     }
+    LOG(prefix<<qname<<": truncated bit set, over UDP"<<endl);
```

Comment:
At a quick glance, it looks like this line could be logged if the query was sent over TCP provided that `dontThrottle` is set?
Actually it's quite weird that we skip the ` truncated bit set, over TCP?` log and the return when `dontThrottle` is set, shouldn't we only skip the throttling?

Reply:
Looks like 5c8c72bf1e4cec59c31445cdc6d97e7669c74669 failed to make that distinction. I'll refactor.

Reply:
refactored, force-pushed